src/EnableNewWebImplementation.ts

```diff
@@ -3,15 +3,28 @@ import { Platform } from 'react-native';
 let useNewWebImplementation = true;
 let getWasCalled = false;
 
+/**
+ * @deprecated legacy implementation is no longer supported. This function will be removed in Gesture Handler 3
+ */
 export function enableExperimentalWebImplementation(
   _shouldEnable = true
 ): void {
   // NO-OP since the new implementation is now the default
+  console.warn(
+    'Legacy web implementation is deprecated. This function will be removed in Gesture Handler 3'
```

Comment:
That's still not what I meant - first sentence just doesn't fit in that case. _**"Legacy web implementation is deprecated. (...)"**_ suggests that someone either tried to enable legacy implementation, or was using it. We should either rephrase it, or leave only second sentence.

Reply:
I agree that might've been the implication.
Rephrased in [ce147b5](https://github.com/software-mansion/react-native-gesture-handler/pull/3259/commits/ce147b52d67b87405395f2071c44fa35f25b919b)